moveit_experimental/moveit_jog_arm/config/ur_simulated_config.yaml

```diff
@@ -7,9 +7,12 @@ use_gazebo: true # Whether the robot is started in a Gazebo simulation environme
 robot_link_command_frame:  world  # commands must be given in the frame of a robot link. Usually either the base or end effector
 command_in_type: "speed_units" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
 scale:
-  linear:  0.6  # Max linear velocity. Meters per publish_period. Units is [m/s]. Only used if command_in_type=="unitless"
-  rotational:  0.3 # Max angular velocity. Rads per publish_period. Units is [rad/s]. Only used if command_in_type=="unitless"
-  joint: 0.3  # Max joint angular/linear velocity. Rads or Meters per publish period. Units is [rad/s] or [m/s].
+  # Scale parameters are only used if command_in_type=="unitless"
+  linear:  0.6  # Max linear velocity. Meters per publish_period. Unit is [m/s]. Only used for Cartesian commands.
+  rotational:  0.3 # Max angular velocity. Rads per publish_period. Unit is [rad/s]. Only used for Cartesian commands.
+  # Max joint angular/linear velocity. Rads or Meters per publish period. Only used for joint commands on joint_command_in_topic.
+  joint: 10
+max_joint_velocity: 1  # Unit is rad/s. Applies to Cartesian commands.
```

Comment:
>  Applies to Cartesian commands.

I don't understand this comment - does it not apply to other types of commands? Does this max get applied at the *input* or *output* of JogArm? How is there a joint limit for a Cartesian command?

Reply:
OK, added more/better comments

Reply:
I'm still confused about how you apply a joint limit to a Cartesian commands though...

Reply:
delta_theta = J^t * delta_x

So it's after you calculate a delta_theta